pages/collection/[chain]/[contract].tsx

```diff
@@ -239,7 +240,15 @@ const CollectionPage: NextPage<Props> = ({ id, ssr }) => {
             <CollectionActions collection={collection} />
           </Flex>
           <StatHeader collection={collection} />
-          <Tabs.Root defaultValue="items">
+          <Tabs.Root
+            defaultValue="items"
+            onValueChange={(value) => {
+              if (value === 'items') {
+                setSize(1)
```

Comment:
If there are more pages loaded, does this actually refresh the pages when you load the second page?

Reply:
I think we need to probably call resetCache here.

Reply:
Yep, you're right 👍